03_011/src/component/card/card.js

```diff
@@ -0,0 +1,20 @@
+import './card.css';
+
+const Card = (information) => {
+  const render = () => {
+    const { idx, title, imageUrl, mediaName, url, summaryContent } =
+      information;
+    return `<div id="category-item-${idx}">
+        <div class="category-item-image" style="background-image : url(${imageUrl})"></div>
+        <div class="category-item-title">${title}</div>
+        <div class="category-item-summary">${summaryContent || '...'}</div>
+        <div class="more-info-container">
+            <div class="category-item-media">by ${mediaName}</div>
+            <button id="bookmark-button-${idx}" class="bookmark-button">☆</button>
+        </div>
+    </div>`;
+  };
+  return render();
```

Comment:
```suggestion
const { idx, title, imageUrl, mediaName, url, summaryContent } = information;
return `
  <div id="category-item-${idx}">
      <div class="category-item-image" style="background-image : url(${imageUrl})"></div>
      <div class="category-item-title">${title}</div>
      <div class="category-item-summary">${summaryContent || '...'}</div>
      <div class="more-info-container">
          <div class="category-item-media">by ${mediaName}</div>
          <button id="bookmark-button-${idx}" class="bookmark-button">☆</button>
      </div>
  </div>
`;
```
굳이 render를 선언할 필요는 없을 것 같아요